src/webgl/material.js

```diff
@@ -476,6 +476,49 @@ p5.prototype.specularMaterial = function(v1, v2, v3, a) {
   return this;
 };
 
+/**
+ * Sets the amount of gloss in the surface of shapes.
+ * Used in combination with specularMaterial() in setting
+ * the material properties of shapes.
+ * @method shininess
+ * @param {Number} shine Degree of Shininess.
+ *                       Defaults to 1.
+ * @chainable
+ * @example
+ * <div>
+ * <code>
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ * }
+ * function draw() {
+ *   background(0);
+ *   noStroke();
+ *   let locX = mouseX - width / 2;
+ *   let locY = mouseY - height / 2;
+ *   ambientLight(60, 60, 60);
+ *   pointLight(255, 255, 255, locX, locY, 50);
+ *   specularMaterial(250);
+ *   translate(-23, 0, 0);
+ *   shininess(1);
+ *   sphere(20);
+ *   translate(46, 0, 0);
+ *   shininess(10);
```

Comment:
I think you can make this number higher to show a larger range of possible effects